tools/chpl-language-server/src/chpl-language-server.py

```diff
@@ -1170,7 +1168,8 @@ def rebuild_index(self):
             # TODO: suppress resolution errors due to false-positives
             # this should be removed once the resolver is finished
             with self.context.context.track_errors() as _:
-                self._search_instantiations(asts)
+                for ast in asts:
+                    self._search_instantiations(ast)
```

Comment:
why is this change needed? doesn't `chapel.each_matching` handle that? it should only yield things of the pattern

Reply:
It's not needed, but I prefer it this way rather than relying on the fact that both types yield the same thing.